### ISSUE TITLE
Action List Modality/Deletion Confirmation

### ISSUE THREAD

**Issue by RobertBColton** [IsmAvatar/LateralGM#165]
When you delete an action, the frame is not closed. This is not a regression and exists in lgm16b4
There are two ways to look at this problem. The user may accidentally delete the action but want to save the action contents. But if we continue to allow the frame to remain open the user may also not notice and continue coding only to find out that their changes are gone. So this is a double edged sword, and fixing it seems to involve choosing the lesser of two evils. However it is also possible that when removing actions from the list we could check to see if they have open frames and warn the user if they want to continue or if they would like to close the frame, I would rather go with a message similar to the latter.

Now another issue to discuss is with the new undo manager for action lists. What exactly should happen if the user performs an undo operation while the action frame is open? Similar to before we can also warn the user and ask if they want to commit changes to the action frame and whether to close the frame before they are allowed to continue the undo operation.
